app/helpers/europeana/i18n/js_translations_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Europeana
+  module I18n
+    module JsTranslationsHelper
+      def js_translation_files
```

Comment:
Why "files" in the method name, and why is the return value an array, when there is only ever one path returned?

Perhaps rename the method something like `i18n_js_asset_path`.